vue/components/ui/organisms/entity-shower/entity-shower.vue

```diff
@@ -28,50 +28,91 @@ export const EntityShower = {
     name: "entity-shower",
     mixins: [partMixin],
     props: {
+        /**
+         * The entity name

```

Comment:
JSDocs - unlike comments - end with a period (`.`).

Reply:
Try by a little more descriptive. What is the entity name? Can it be `João` or should it be `Person`? Use examples if  necessary.